src/libraries/System.Drawing.Common/src/System/Drawing/Drawing2D/Matrix.cs

```diff
@@ -93,6 +116,30 @@ public float[] Elements
             }
         }
 
+        /// <summary>
+        ///  Gets/sets the elements for the matrix.
+        /// </summary>
+        public unsafe Matrix3x2 MatrixElements
+        {
+            get
+            {
+                Matrix3x2 matrix = default;
+                Gdip.CheckStatus(Gdip.GdipGetMatrixElements(new HandleRef(this, NativeMatrix), (float*)&matrix));
+                return matrix;
+            }
+            set
+            {
+                Gdip.CheckStatus(Gdip.GdipSetMatrixElements(
```

Comment:
So GDI exposes a GetMatrixElements that takes a single pointer, but a SetMatrixElements that takes each float individually?

Reply:
Just triple checked. While `GpMatrix` has a constructor that takes an array of points, it isn't projected into the flat api or the public `Matrix` class. 😞 I could create the identity matrix and set all the points at once, but I presume that would be much slower.